specifyweb/frontend/js_src/lib/localization/query.ts

```diff
@@ -875,4 +875,10 @@ export const queryText = createDictionary({
     'fr-fr': 'Faux ou vide',
     'ru-ru': 'Ложь или пусто',
   },
+  viewListOfIds: {
+    'en-us': 'View list of ids.',
```

Comment:
remove trailing dot? while not a big issue, we normally don't have it in the text that is used in `titles` (just for consistency, and to allow reusing title strings outside of titles)

```suggestion
    'en-us': 'View list of ids',
```